src/hooks/useActiveDescendantListFocus/__tests__/useActiveDescendantListFocus.jest.js

```diff
@@ -5,32 +5,95 @@ import userEvent from "@testing-library/user-event";
 let element;
 const FIRST_ITEM_ID = "id-1";
 const SECOND_ITEM_ID = "id-2";
-const ITEM_IDS = [FIRST_ITEM_ID, SECOND_ITEM_ID];
+const THIRD_ITEM_ID = "id-3";
+const FOURTH_ITEM_ID = "id-4";
+const FIFTH_ITEM_ID = "id-5";
+const ITEM_IDS = [FIRST_ITEM_ID, SECOND_ITEM_ID, THIRD_ITEM_ID, FOURTH_ITEM_ID, FIFTH_ITEM_ID];
 
-function renderHookForTest({ onItemClick = jest.fn(), isItemSelectable = () => true, isHorizontal = false }) {
+function renderHookForTest({
+  onItemClick = jest.fn(),
+  defaultVisualFocusFirstIndex = false,
+  isItemSelectable = () => true,
+  isHorizontal = false
+}) {
   element = document.createElement("div");
   element.tabIndex = -1; // some tests focus the element - a tabIndex value is required for updating the document.activeIndex value
   document.body.appendChild(element);
 
-  return renderHook(() =>
-    useActiveDescendantListFocus({
-      focusedElementRef: {
-        current: element
-      },
-      itemsIds: ITEM_IDS,
-      isItemSelectable: isItemSelectable,
-      onItemClick,
-      isHorizontalList: isHorizontal
-    })
-  );
+  const props = {
+    focusedElementRef: {
+      current: element
+    },
+    defaultVisualFocusFirstIndex,
+    itemsIds: ITEM_IDS,
+    isItemSelectable: isItemSelectable,
+    onItemClick,
+    isHorizontalList: isHorizontal
+  };
+  return renderHook(argprops => useActiveDescendantListFocus({ ...props, ...argprops }));
 }
 
-function runListUnitTest(isHorizontal) {
+function runListUnitTest({ isHorizontal, defaultVisualFocusFirstIndex }) {
   const moveForwardKey = isHorizontal ? "{arrowRight}" : "{arrowDown}";
   const oppositeMoveForwardKey = !isHorizontal ? "{arrowRight}" : "{arrowDown}";
+
+  it("should focus index + 1 item when user press keyboard forward", async () => {
+    const onItemClick = jest.fn();
+    const { result } = renderHookForTest({ onItemClick, isHorizontal, defaultVisualFocusFirstIndex });
+
+    act(() => {
+      // set focus on the list's element which in charge on natural focus element
+      element.focus();
+    });
+
+    act(() => {
+      // move visual focus to first item
+      userEvent.keyboard(moveForwardKey);
+    });
+    let before = result.current.visualFocusItemIndex;
+    const keyboardTimes = 2;
+
+    for (let i = 0; i < keyboardTimes; i++) {
+      act(() => {
+        // move visual focus to first item
+        userEvent.keyboard(moveForwardKey);
+      });
+    }
```

Comment:
nitpick - I think that calling the `act` twice is more readable than a for loop..